pkg/windows/windows.go

```diff
@@ -474,7 +488,16 @@ func (vm *windows) Configure() error {
 		return errors.Wrapf(err, "error configuring containerd")
 	}
 
-	return vm.runBootstrapper()
+	if err := vm.ensureWICDSecretContent(credentials); err != nil {
+		return err
+	}
+	wicdBootstrapCmd := fmt.Sprintf("%s bootstrap --desired-version %s --api-server %s --sa-ca %s --sa-token %s",
+		wicdPath, desiredVer, apiServerURL, wicdCAFile, wicdTokenFile)
```

Comment:
Could `desiredVer` and/or `apiServerURL` be empty? Is it safe passing an empty value, i.e. `--desired-version ` ?